SU-SWS/stanford_basic
# D8CORE-3441: a11y changes for dropdowns

# READY FOR REVIEW

# Summary
- A11y changes for dropdowns.
- Adds a hover state to the `su-nav-toggle` component.
- Switches the arrow from red to black.

# Review By (Date)
- code freeze

# Urgency
- needed for 2.0.0

# Steps to Test

1. Checkout branch
2. clear caches
3. create drop down menu items in the navigation menu.
4. Observe the state of the carets in the navigation menu as you hover and click.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- D8CORE-3441

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
